binding/Binding/SKBitmap.cs

```diff
@@ -718,7 +718,10 @@ public bool PeekPixels (SKPixmap pixmap)
 			if (pixmap == null) {
 				throw new ArgumentNullException (nameof (pixmap));
 			}
-			return SkiaApi.sk_bitmap_peek_pixels (Handle, pixmap.Handle);
+			var result = SkiaApi.sk_bitmap_peek_pixels (Handle, pixmap.Handle);
+			if (result)
+				Referenced (pixmap, this);
```

Comment:
The GC sometimes likes to collect the bitmap in use by a pixmap. This case is important because the bitmap OWNS the pixmap, and the reference is not counted. So a disposal destroys the pixmap.